README.md

```diff
@@ -1,3 +1,27 @@
 # Chartify
 
 Turn your written knitting pattern into a handsome chart!
+
+Chartify takes input in the form of allowed stitches, seperated by line breaks (eventually user input), and returns an ASCII chart.
+
+For example, input the string "k1 k1 p1 p1 k1 k1 \n p1 p1 k1 k1 p1 p1," and Chartify will crate this pattern:
+
+┌─┬─┬─┬─┬─┬─┐  
+│-│-│ │ │-│-│
+├─┼─┼─┼─┼─┼─┤
+│ │ │-│-│ │ │
+└─┴─┴─┴─┴─┴─┘
+
+
+Current supported stitches/input, and how they are rendered:
+
+| Stitch     | Rendering |
+| ----------- | ----------- |
+| k1      | blank box       |
+| p1   | -        |
+| ssk   | \        |
+| k2tog   | /        |
+| yo   | o        |
+| m1   | m        |
+
+
```

Comment:
Some important information to add:

What version of language(s) does this project require? How would a new developer joining the team install this project on their machine? How would they run the tests? How would they run the app itself? A good way to test your README is to ask someone (e.g. one of your mentor team) to go through the steps described in the README and see if they can install, run tests, and run the app.